src/main/handlers/helpers/exportUtils.ts

```diff
@@ -25,3 +25,8 @@ export const getExportFilePath: (
 
   return dialogResponse.filePath as string;
 };
+
+export const fracFpsToDec: (fracFps: string) => number = (fracFps) => {
+  const nums = fracFps.split('/').map((item: string) => +item); // get the numbers (separated by / ), and convert to numbers.
```

Comment:
`Number(item)`

`+item` is a bit 'clever' whereas `Number(item)` is very explicit

Reply:
Changed it to be something a little bit more declarative, 
`.map(Number)`
still a bit iffy in terms of keeping consistent with any other instances of a cast map in the codebase tho.

Reply:
`.map(Number)` is fine :)